Angles/Homework2/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+
+namespace Math
+{
+    class Math
```

Comment:
Неудачное название класса, поскольку оно совпадает с именем класса Math из пространства имен System. Почему бы не оставить стандартное название?
```suggestion
    class Program
```
Обычно так называют класс, содержащий основной метод программы.